src/generic/grid.cpp

```diff
@@ -5914,10 +6066,17 @@ void wxGrid::OnKeyDown( wxKeyEvent& event )
                         case WXGRID_CURSOR_RESIZE_ROW:
                         case WXGRID_CURSOR_RESIZE_COL:
                             // reset to size from before dragging
-                            (m_cursorMode == WXGRID_CURSOR_RESIZE_ROW
-                                ? static_cast<const wxGridOperations&>(wxGridRowOperations())
-                                : static_cast<const wxGridOperations&>(wxGridColumnOperations())
-                            ).SetLineSize(this, m_dragRowOrCol, m_dragRowOrColOldSize);
+                            if ( m_dragRowOrCol == -2 )
+                            {   if ( m_cursorMode == WXGRID_CURSOR_RESIZE_ROW )
```

Comment:
Some strange indentation here
```suggestion
                            {
                                if ( m_cursorMode == WXGRID_CURSOR_RESIZE_ROW )
```